pkg/network/protocols/ebpfpostgres/types.go

```diff
@@ -1,18 +1,20 @@
 // Unless explicitly stated otherwise all files in this repository are licensed
 // under the Apache License Version 2.0.
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
-// Copyright 2016-present Datadog, Inc.
+// Copyright 2024-present Datadog, Inc.
 
 //go:build ignore
 
-package postgres
+package ebpfpostgres
 
 /*
 #include "../../ebpf/c/protocols/postgres/types.h"
 #include "../../ebpf/c/protocols/classification/defs.h"
 */
 import "C"
 
+// This package was created to avoid cyclic imports.
```

Comment:
Which cycle is this preventing? I don't see any imports.

Reply:
The cyclic dependency that will be prevented is represented in this [PR](https://github.com/DataDog/datadog-agent/pull/27110). I will add the ability to use a configuration value for the postgres telemetry buffer size, and I want to use the same buffer size value. When I do so, it will create cyclic imports between the config package and the postgres package.